lv_conf_template.h

```diff
@@ -294,14 +294,20 @@
 #endif
 
 /** Use NXP's G2D on MPU platforms. */
-#define LV_USE_DRAW_G2D 0
+#define LV_USE_G2D 0
+
+#if LV_USE_G2D
+    /** Use G2D for drawing. **/
+    #define LV_USE_DRAW_G2D 1
+
+    /** Use G2D to rotate display. **/
+    #define LV_USE_ROTATE_G2D 0
```

Comment:
```suggestion
    #define LV_USE_G2D_ROTATE 0
```

We add the "module" name first.

Reply:
If so, please change to `LV_USE_PXP_ROTATE` as well, for consistency.

Reply:
@kisvegabor But I see inconsistency between `LV_USE_DRAW_G2D` and `LV_USE_G2D_ROTATE`.
While `LV_USE_DRAW_G2D` is consistent with `LV_USE_DRAW_SW`...
So, how shall it be?